packages/cli/src/License.ts

```diff
@@ -79,6 +84,15 @@ export class License {
 		}
 	}
 
+	async collectUsageMetrics() {
```

Comment:
We may want to add number of executions as well?

Reply:
this is currently not a requirement.